data-upgrade-pages/src/main/resources/conf/portal/configuration.xml

```diff
@@ -156,5 +156,52 @@
       </init-params>
     </component-plugin>
   </external-component-plugins>
+
+  <external-component-plugins>
+    <target-component>org.exoplatform.commons.upgrade.UpgradeProductService</target-component>
+    <component-plugin>
+      <name>DlpQuarantinePagesUpgradePlugin</name>
+      <set-method>addUpgradePlugin</set-method>
+      <type>org.exoplatform.agenda.migration.PagesMigration</type>
```

Comment:
I think this need to be renamed (Removing agenda from the package)

Reply:
+1